src/status_im/contexts/profile/login/events.cljs

```diff
@@ -97,11 +97,10 @@
 
                   (or pairing-completed? (get db :onboarding/new-account?))
                   [[:dispatch [:onboarding/finalize-setup]]
-                   [:dispatch [:onboarding/finish-onboarding false]]]
+                   [:dispatch [:onboarding/finish-onboarding]]]
 
                   :else
-                  [[:dispatch [:update-theme-and-init-root :screen/shell-stack]]
-                   [:dispatch [:profile/toggle-testnet-mode-banner]]]))})))
+                  [[:dispatch [:shell/show-root-view]]]))})))
```

Comment:
Here we're introducing a new event for representing when we want to show root shell-stack of the app. This is being introduced because we needed a common place to check whether someone has been prompted with the screen for the notification permissions, and we needed to cover all the possible ways someone could arrive to the app including:
* login
* signup
* recovery
* syncing